Makefile

```diff
@@ -344,6 +326,7 @@ clean: kustomize-external-go-plugin-clean
 	rm -f $(MYGOBIN)/pluginator
 	rm -f $(MYGOBIN)/kustomize
 	rm -f $(MYGOBIN)/golangci-lint-kustomize
+	rm -f $(MYGOBIN)/helm
```

Comment:
Maybe not.  Best to leave deleting helm to a distinct target (or no target at all).

The Makefile should only delete the things that it knows that it installed;
we could track that, but it requires more state and isn't worth it